Makefile

```diff
@@ -249,6 +249,8 @@ $(BUILD_DIR)/images/%/$(DUMMY):
 		--build-arg TARGETARCH=$(ARCH) \
 		--build-arg TARGETOS=linux \
 		$(BUILD_ARGS) \
+		--cache-to type=inline,mode=max \
+		--cache-from type=registry,ref=$(DOCKER_NS)/fabric-$*:$(FABRIC_VER) \
```

Comment:
When building FABRIC_VER (e.g. `3.0.0`) image, a prior FABRIC_VER `3.0.0` will not exist in the registry yet. So I'm not understanding how the cache would work. Can you explain?

Reply:
First, time it won't have, while once pushed, the rest of the builds will utilize cached layers if no changes have been made to the layers. Here some more details: https://lipanski.com/posts/speed-up-your-docker-builds-with-cache-from

Reply:
Well, as a matter of fact, I wanted to see whether enabling cache is worth it.